ksqldb-engine/src/main/java/io/confluent/ksql/engine/KsqlEngine.java

```diff
@@ -128,10 +130,16 @@ public int numberOfLiveQueries() {
     return primaryContext.getPersistentQuery(queryId);
   }
 
+  @Override
   public List<PersistentQueryMetadata> getPersistentQueries() {
     return ImmutableList.copyOf(primaryContext.getPersistentQueries().values());
   }
 
+  @Override
+  public List<QueryMetadata> getAllLiveQueries() {
+    return ImmutableList.copyOf(new ArrayList<>(allLiveQueries));
```

Comment:
```suggestion
    return ImmutableList.copyOf(allLiveQueries);
```
the `new ArrayList` is not necesasry, `copyOf` takes an `Iterable`